JMMServer/JMMServiceImplementationKodi.cs

```diff
@@ -639,6 +640,9 @@ private System.IO.Stream GetGroupsFromFilter(int userid, string GroupFilterId)
                                         }
                                         v.OriginalTitle = v.OriginalTitle.Substring(0, v.OriginalTitle.Length - 1);
                                         //proper naming end
+                                        List<AnimeSeries> sers = grp.GetAllSeries();
+                                        AnimeSeries ser = sers[0];
+                                        v.totalLocal = ser.GetAnimeEpisodesCountWithVideoLocal();
```

Comment:
Yeah, I don't know kover doesn't show the unwatched numbers at every level, so it's kind of hard to test. It wasn't a system error actually. I misread one of my test queries. It was a typo in Nakamori. Made a PR for it.